flutter-studio/src/io/flutter/android/AndroidModuleLibraryType.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.android;
+
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.roots.libraries.LibraryType;
+import com.intellij.openapi.roots.libraries.NewLibraryConfiguration;
+import com.intellij.openapi.roots.libraries.PersistentLibraryKind;
+import com.intellij.openapi.roots.libraries.ui.LibraryEditorComponent;
+import com.intellij.openapi.roots.libraries.ui.LibraryPropertiesEditor;
+import com.intellij.openapi.vfs.VirtualFile;
+import icons.FlutterIcons;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+
+public class AndroidModuleLibraryType extends LibraryType<AndroidModuleLibraryProperties> {
+  public static final String LIBRARY_NAME = "Android Libraries";
```

Comment:
<img width="288" alt="screen shot 2018-07-10 at 11 54 51 am" src="https://user-images.githubusercontent.com/8518285/42531173-2571cb2e-8438-11e8-967d-3de868fc0366.png">

Reply:
I see the latest is `Android Libraries`. It could still be useful to give a hint where these libraries come from (the Flutter plugin dependencies), but YMMV.